e2e/tests/ui-driven/src/mocks/geospatialMocks.ts

```diff
@@ -37,3 +37,165 @@ export const mockMapGeoJson: Feature = {
     "organisation-entity": "13",
   },
 };
+
+export const mockPropertyConstraints = {
```

Comment:
Seemed like an overzealous mock, but seemed the past way to ensure we get all the data pulled in that we need for showing constraints. Happy to refine if we think it is required

Reply:
I don't think this is a complete mock - I would expect it to additionally include `metadata` from Planning Data (like you've included in the roads mock)